pandas/indexes/multi.py

```diff
@@ -656,9 +659,58 @@ def _has_complex_internals(self):
         return True
 
     @cache_readonly
+    def is_monotonic(self):
+
+        # TODO
+        # this is unfortunate we end up tupelizing
+        # just to determine monotonicity :<
+
+        # fast-path
+        if not self.levels[0].is_monotonic:
+            return False
+
```

Comment:
If someone comes up with a 'nice' way to do this (from a list of arrays), would be great.